pkg/controller/bypass/bypass_controller.go

```diff
@@ -190,6 +293,16 @@ func checkSidecar(client kubernetes.Interface, pod *corev1.Pod) (bool, error) {
 	return false, nil
 }
 
+func checkKmesh(pod *corev1.Pod) (bool, error) {
```

Comment:
Is the error here a redundant return value?

Reply:
> Is the error here a redundant return value?

done